packages/dhis2/src/Adaptor.js

```diff
@@ -781,6 +880,34 @@ export function dv(dataElement, value) {
   return { dataElement, value };
 }
 
+export function shouldUseNewTracker(resourceType) {
+  return (
+    resourceType === 'trackedEntityInstances' ||
+    resourceType === 'tracker' ||
+    resourceType === 'tracker/trackedEntities' ||
```

Comment:
Hmm I think the logic is backwards here.

If the resourceType is `tracker/`, `trackedEntityInstance, `enrollments, `events` or `relationships`, we should use the new tracker. OTHERWISE we use the old API.

A few unit tests on this function would be super helpful so that we can easily see how it works! They're easy to create as well because we don't need to mock anything.

Reply:
@josephjclark  in this comment by the old api you mean the older versions of dhis2 for `tracker`?

Reply:
Yep, exactly!